src/docs/product/issues/performance-issues/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: "Performance Issues"
+sidebar_order: 15
+description: "Learn more about Performance Issues and how to use them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+The **Issue Details** page for performance issues helps you to gain further insight into the source of the issue and the impact it has on your application's users. The main area of the page displays information about a specific transaction event that's part of an issue. The top panel and right-hand sidebar of the page display a summary of all the events grouped together in this issue.
+
+<Note>
+
+You can find performance issues by searching `issue.category:performance` in **Issues**, **Discover** and **Dashboards**.
+
+</Note>
+
+![Performance issue](performance-issue.png)
+
+While the **Issue Details** page displays information about a specific transaction event that's part of an issue, you can navigate between the events of an issue using the "Older" and "Newer" buttons. You'll see information about the issue as well as the details of the most recent event. You'll be able to see the summary of when the issue was seen, a breakdown of the tags, and breadcrumbs if they are available.
+
+## How Performance Issues Work
+
+Sentry detects performance issues by scanning incoming transaction events. It checks various properties of the transaction (span durations, span arrangement, span types, etc.) to detect likely problems. Sentry then checks these properties against corresponding thresholds to determine whether a performance issue exists. Sentry then generates a unique problem fingerprint based on the problem type and location in the application, and uses that fingerprint to create a performance issue.
+
+## Tags
+
+Tags are key/value string pairs that are both indexed and searchable. For example, a tag provides you with information such as the browser, device, or user associated with the event. Tags are the diagnostic information sent by the SDK for the individual event.
+
```

Comment:
Tags are described in its own section as well https://docs.sentry.io/platforms/javascript/enriching-events/tags/#:~:text=Tags%20are%20key%2Fvalue%20string,for%20a%20set%20of%20events

Wondering if it would make sense we can share the definition or link one to another.

Reply:
I'm fine with using the definition from the SDK docs if you want to @gggritso

Reply:
Hm, definitely maybe! I like the definition in the SDK docs, but the definition here is better at explaining the significance of tags in this specific context 🤔